src/controllers/group/index.js

```diff
@@ -50,3 +50,28 @@ const createProductIfNotGiven = async (input, models, { transaction }) => {
 
 const savePhotos = (productId, photos, { transaction }) => productPhoto
   .bulkCreate(photos.map((url) => ({ productId, url })), { transaction })
+
+export const deleteGroup = async (obj, { id }, context) => {
+  const group = await context.models.group.findByPk(id)
+
+  if (!group || await isNotDeleteable(group, context)) return 0
```

Comment:
What do you think of changing `isNotDeleteable` to `canGroupBeDeleted`? I think it sounds better in English :slightly_smiling_face: